Client/Frontend/Strings.swift

```diff
@@ -38,7 +38,7 @@ extension Strings {
 
 // Activity Stream.
 extension Strings {
-    public static let ASPageControlButton = NSLocalizedString("AS.PageControl.Button", comment: "The page control button that lets you switch between pages in top sites")
+    public static let ASPageControlButton = NSLocalizedString("AS.PageControl.Button", value: "Next Page", comment: "The page control button that lets you switch between pages in top sites")
```

Comment:
The localization ID is a clue to help localizers determine the context of where this string is used, so "AS" might not be very helpful since localizers won't know what that stands for. I see we already have a number of other string IDs with "AS" already, though, so hopefully they'll be able to figure it out.